pkg/constants/metadata.go

```diff
@@ -39,13 +39,13 @@ const (
 	// The full annotation name is supposed to be "<routingClass>.routing.controller.devfile.io/<anything>"
 	RoutingAnnotationInfix = ".routing.controller.devfile.io/"
 
-	// DevWorkspaceStorageTypeLabel defines the strategy used for provisioning storage for the workspace.
+	// DevWorkspaceStorageTypeAtrr defines the strategy used for provisioning storage for the workspace.
 	// If empty, the common PVC strategy is used.
 	// Supported options:
 	// - "common": Create one PVC per namespace, and store data for all workspaces in that namespace in that PVC
 	// - "async" : Create one PVC per namespace, and create a remote server that syncs data from workspaces to the PVC.
 	//             All volumeMounts used for devworkspaces are emptyDir
-	DevWorkspaceStorageTypeLabel = "controller.devfile.io/storage-type"
+	DevWorkspaceStorageTypeAtrr = "controller.devfile.io/storage-type"
```

Comment:
I wonder if storage impl is common enough that we can move it to devfile/api side, where each implementation may support or does not the configured implementation.
But I'm fine with moving it out of the current PR scope.

Reply:
I think moving it would mean that if we want to add another storage type for our use-case, we'd have to first do it on the devfile/api side. I think I prefer it scoped to the operator (as it's a feature primarily for development) -- I think odo treats volumes differently than we do.